dev/integration_tests/android_splash_screens/splash_screen_kitchen_sink/ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think we can either not name this whole project android_splash_screen (we can add ios parts to it) or if it still builds, just remove the ios directory from this project

Reply:
I deleted all the iOS directories for the projects.